wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/wss/impl/misc/SecurityUtil.java

```diff
@@ -66,6 +67,9 @@
 import com.sun.xml.ws.security.SecurityTokenReference;
 
 import org.w3c.dom.Node;
+
+import static com.sun.xml.wss.provider.wsit.logging.LogStringsMessages.WSITPVD_0066_SERVLET_CONTEXT_NOTFOUND;
```

Comment:
Why are static imports an issue in 2022? They improve code readability since Java 5 (2004). I would rather expect that someone would remove all those trailing spaces, mixed tabs and spaces, TODOs and commented out code, fix broken formatting, cyclic dependencies between packages ... so much maintenance to do in many JEE projects ...

But if you really want it, you got it ... in the morning :-)

Reply:
> (there is currently none in the whole code base...)

Actually this is not true, there are even static imports with asterisks, where I would agree it is not ok:
![image](https://user-images.githubusercontent.com/302904/157634296-b1c3db2b-ad72-42ab-88ec-2c61b063b18d.png)

Reply:
right, don't know how the wrong part the comment got there. I swear it was deleted before the comment button was hit....

Anyway - all these got into the codebase with the comment "remove static imports" but as can be seen, people usually did not care and reason "someone else will do it" led to where we are (the same applies to tabs/spaces, formatting issues, etc....)

Reply:
And another issue - in the SecurityUtil  is used several other LogStringMessages classes, one is in imports, other are used with full class name. Usually I don't need to know the full class name, I need just to be able to get to the message. Even Eclipse can go through using F3. Automatic optimization of imports doesn't break it too. So I would really prefer this

`log.log(Level.WARNING, WSITPVD_0066_SERVLET_CONTEXT_NOTFOUND());`

over
            
```
log.log(Level.WARNING,                
   com.sun.xml.wss.provider.wsit.logging.LogStringsMessages.WSITPVD_0066_SERVLET_CONTEXT_NOTFOUND());
```

Reply:
all of those are coming from the wrong resource bundles in the first place. You moved the method but not the message it uses between resources bundles.

Correct thing to do is to move the message from `com.sun.xml.wss.provider.wsit.logging.LogStrings` to `com.sun.xml.wss.impl.misc.LogStrings` resource (in all language variations). If you do that for other two messages, no extra import is going to be required as the file will be in the same package

Reply:
Seems even ReflectionUtil then could be renamed to ServletContextUtil or something similar ...

Reply:
No.

the message with the key `WSITPVD_0066_SERVLET_CONTEXT_NOTFOUND` was used in `com.sun.xml.wss.provider.wsit.SecurityTubeFactory` (and it's the only usage I've found). After your change, this key is used in the `com.sun.xml.wss.impl.misc.SecurityUtil` and the resource bundle for this class is `com.sun.xml.wss.impl.misc.LogStrings`, so the `WSITPVD_0066_SERVLET_CONTEXT_NOTFOUND` 
 message should be moved as well (`com.sun.xml.wss.impl.misc.LogStringsMessages` is created from the resource file by the build)

Reply:
basically move https://github.com/eclipse-ee4j/metro-wsit/blob/master/wsit/ws-sx/wssx-impl/src/main/resources/com/sun/xml/wss/provider/wsit/logging/LogStrings.properties#L130 to https://github.com/eclipse-ee4j/metro-wsit/blob/master/wsit/ws-sx/wssx-impl/src/main/resources/com/sun/xml/wss/logging/impl/misc/LogStrings.properties

Reply:
That will no help readability too, because LogStringMessages in imports lead to "crypto" package, not to "misc".
So let's do both (own class for SevletContext wrapping related things  in next PR). I was already digging around, seems that the "zone" is quite small and can be in a single class, I can do that in an hour.

Reply:
right, as you are in this, is moving https://github.com/eclipse-ee4j/metro-wsit/blob/master/wsit/ws-sx/wssx-impl/src/main/resources/com/sun/xml/wss/logging/impl/crypto/LogStrings.properties#L31 and https://github.com/eclipse-ee4j/metro-wsit/blob/master/wsit/ws-sx/wssx-impl/src/main/resources/com/sun/xml/wss/logging/LogStrings.properties#L1886 to the same resource file to avoid conflicts in imports not an option?